applications/GeoMechanicsApplication/tests/test_partially_saturated/test_saturated_below_phreatic_level_pw_triangle6N/column.mdpa

```diff
@@ -0,0 +1,393 @@
+       18          0    20    19    17    61    31    60
+       19          0    19    20    22    61    42    62
+       20          0    22    21    19    63    32    62
+End Elements
+
+Begin SubModelPart Parts_Solid_Grond // Group Grond // Subtree Parts_Solid
+    Begin SubModelPartNodes
+            1
+            2
+            3
+            4
+            5
+            6
+            7
+            8
+            9
+           10
+           11
+           12
+           13
+           14
+           15
+           16
+           17
+           18
+           19
+           20
+           21
+           22
+           23
+           24
+           25
+           26
+           27
+           28
+           29
+           30
+           31
+           32
+           33
+           34
+           35
+           36
+           37
+           38
+           39
+           40
+           41
+           42
+           43
+           44
+           45
+           46
+           47
+           48
+           49
+           50
+           51
+           52
+           53
+           54
+           55
+           56
+           57
+           58
+           59
+           60
+           61
+           62
+           63
+    End SubModelPartNodes
+    Begin SubModelPartElements
+            1
+            2
+            3
+            4
+            5
+            6
+            7
+            8
+            9
+           10
+           11
+           12
+           13
+           14
+           15
+           16
+           17
+           18
+           19
+           20
+    End SubModelPartElements
+End SubModelPart
+
+Begin SubModelPart gravity
+    Begin SubModelPartNodes
+            1
+            2
+            3
+            4
+            5
+            6
+            7
+            8
+            9
+           10
+           11
+           12
+           13
+           14
+           15
+           16
+           17
+           18
+           19
+           20
+           21
+           22
+           23
+           24
+           25
+           26
+           27
+           28
+           29
+           30
+           31
+           32
+           33
+           34
+           35
+           36
+           37
+           38
+           39
+           40
+           41
+           42
+           43
+           44
+           45
+           46
+           47
+           48
+           49
+           50
+           51
+           52
+           53
+           54
+           55
+           56
+           57
+           58
+           59
+           60
+           61
+           62
+           63
+    End SubModelPartNodes
+End SubModelPart
+
+Begin SubModelPart DISPLACEMENT_Bottom // Group Bottom // Subtree DISPLACEMENT
+    Begin SubModelPartNodes
+            1
+            2
+           43
+    End SubModelPartNodes
+End SubModelPart
+
+Begin SubModelPart DISPLACEMENT_Sides // Group Sides // Subtree DISPLACEMENT
```

Comment:
And also this sub-model part is not being used.

Reply:
Removed